Tools/autotest/autotest.py

```diff
@@ -426,18 +426,17 @@ def run_tests(steps):
                 passed = False
                 failed.append(step)
                 results.add(step, '<span class="failed-text">FAILED</span>', time.time() - t1)
-                continue
+            else:
```

Comment:
Sure, can do that, I'd like to point out I just kept the old structure though. And what do you mean by make run_step void? You mean make it return nothing or make it take no parameters? Personally I don't think either is a good idea, since it'll involve reading and setting more file level variables, which makes the code inevetably more difficult to understand, test and debug...